src/Infrastructure/Project/Table/ProjectTableFactory.ts

```diff
@@ -23,6 +24,7 @@ export class ProjectTableFactory {
         .createBuilder()
         .value(project.name)
         .value(project.customer.name)
+        .value(project.active ? '✅ ' : '🚫 ')
```

Comment:
C'est probablement joli ces emojis, mais en prenant en compte l'accessibilité j'aurais peut-être vu plutôt d'écrire "Oui" quand c'est actif et laisser vide quand ça l'est pas ?

```suggestion
        .value(project.active ? 'Oui ' : ' ')
```

Reply:
Apr1es réflexion, je trouve quand même que ça améliore la lisibilité les pictos, et donc quelque part cela améliore l'accessibilité, en tout cas pour moi qui est pas une super bonne vue c'est un confort supplémentaire pour identifier les projets actifs. C'est surtout un cas d'étude car on ne modifie pas souvent les projets, mais du coup j'ai ajouter un nouveau type de cellule de tableau pour intégrer le picto sous forme de classe css et donc ne pas impacter la sémantique du code HTML